SparkPost/2web2ui
# TR-1898 - Saved Indicator now updates when sending a preview email

[TR-1898](https://jira.int.messagesystems.com/browse/TR-1898)

### What Changed
- `setHasSaved` invoked when the template updates
- Updated loading state to use a standard component
- Deleted custom styles

### How To Test
- Go to `/templatesV2`
- Click on a template
- In draft mode, type something in to the editor and notice that the top right reads "Unsaved Changes"
- Click on the "Send Test Email" button and see that the top right reads as "Saved"

### To Do
- [x] Incorporated feedback
